modules/ting_covers_ddb/includes/ting_covers_ddb.admin.inc

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file
+ * Administration interface for the ting_covers_ddb module.
+ */
+
+/**
+ * Administration form for the module.
+ *
+ * @ingroup forms
+ *
+ * @see system_settings_form()
+ */
+function ting_covers_ddb_admin_settings_form($form, &$form_state) {
+  $form['ddb'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('DDB Cover Service'),
+    '#description' => t('Fetch covers form <a href="https://cover.dandigbib.org/">DDB Cover Service</a>. It requires that you have setup and configured the client id/secret in adgangsplatformen module.'),
+    '#tree' => FALSE,
+  );
+
+  $form['ddb']['ting_covers_ddb_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Service URL'),
+    '#description' => t('URL to the Additional Information webservice, e.g. http://cover.dandigbib.org'),
+    '#required' => TRUE,
+    '#default_value' => _ting_covers_ddb_get_url(),
+  );
+
+  $form['ddb']['ting_covers_ddb_enable_logging'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Enable logging'),
+    '#default_value' => variable_get('ting_covers_ddb_enable_logging', FALSE),
+  );
+
+  $form['ddb']['development'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Development settings'),
+    '#description' => t('The module request that you have a valid adgangsplatform setup to get an agency token. For testing you can here set an token (which will expire) if you are on an system without Adgangsplatformen enabled. Should <strong>never</strong> be used in production.'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+
+  $form['ddb']['development']['token'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Agency token'),
+    '#description' => t('This should be an agency token generated by using the agency af user and password with @ prefixed which you can generate at <a href="https://openplatform.dbc.dk/v3/" target="_blank">https://openplatform.dbc.dk/v3/</a>'),
```

Comment:
`t('This should be an agency token generated by using the agency af user and password with @ prefixed which you can generate at <a href="https://openplatform.dbc.dk/v3/" target="_blank">https://openplatform.dbc.dk/v3/</a>'),`
->
`t('This should be an agency token generated by using your agency as both user and password with @ prefixed. Token can generated at <a href="https://openplatform.dbc.dk/v3/" target="_blank">https://openplatform.dbc.dk/v3/</a>'),`

Reply:
Updated the comment.